python/src/aiconfig/editor/server/server.py

```diff
@@ -54,26 +120,42 @@ def test():
     return {"key": 6}
 
 
-def run_backend_server(edit_config: EditConfig) -> Result[int, str]:
+@app.route("/api/run")
+async def run():
+    ss = _get_server_state(app)
+    result = await ss.aiconfig_runtime.run(
+        "gen_packing_list",
+        #
+        params={"location": "central park"},
+    )
+    text = result.data[0]
+    return f"<p>{text}</p>"
```

Comment:
Is this hardcoded just for testing? Also note that `run` can't just return `<p>` -- it should return the full output object, plus the AIConfig object -- cc @rossdanlm

Reply:
yes testing, I'll fix this later